aiocache/base.py

```diff
@@ -429,6 +429,25 @@ def plugins(self, value):
     async def _raw(self, command, *args, **kwargs):
         raise NotImplementedError()
 
+    @API.register
+    @API.aiocache_enabled()
+    @API.timeout
+    @API.plugins
```

Comment:
Can you remove `register`, `aiocache_enabled` and `plugins` decorators? Those decorators are for cache specific commands